rpcs3/Emu/Cell/Modules/cellGcmSys.cpp

```diff
@@ -142,7 +142,7 @@ vm::ptr<CellGcmReportData> cellGcmGetReportDataAddressLocation(u32 index, u32 lo
 		cellGcmSys.error("cellGcmGetReportDataAddressLocation: Wrong local index (%d)", index);
 	}
 
-	return vm::ptr<CellGcmReportData>::make(g_fxo->get<gcm_config>()->gcm_info.label_addr + 0x1400 + index * 0x10);
+	return vm::ptr<CellGcmReportData>::make(g_fxo->get<gcm_config>()->gcm_info.label_addr + ::offset32(&RsxReports::report) + index * 0x10);
```

Comment:
Obviously the cellGcmSys changes do not belong in this commit.

Reply:
hmm its a clean up for similar use case in rsx::get_address, so might as well clean it all.

Reply:
My complaint is because its a completely different module (fw) vs the other stuff which is unrelated and not a cleanup. get_address change is not cosmetic, it is functional and adds logging. Changes to gcmsys are doing something different, it would have been better to split. It actually makes more sense to merge the changes to gcmsys with the commit that follows this one and is very easy to do with a soft reset and file reselection.